provisioning/virtual-machines/README-VM-create.md

```diff
@@ -61,7 +61,7 @@ The tools that are required by CrownLabs to work and are automatically installed
 
 1. Customize the basic information at the beginning of the [setup-crownlabs-vm.sh](setup-crownlabs-vm.sh) script (e.g. username and password, address of the docker registry).
 
-2. Execute `./setup-crownlabs-vm.sh <vm-name> create (--no-guest-additions)` to automatically create a new VM, install the Xubuntu OS and (optionally) the Virtualbox Guest Additions.
+2. Execute `./setup-crownlabs-vm.sh <vm-name> create [desktop|server] <ubuntu-version> (--install-guest-additions)` to automatically create a new VM, install the Ubuntu OS and (optionally) the Virtualbox Guest Additions. It is necessary to select a distribution between *Ubuntu Desktop* (i.e. `xubuntu`) and *Ubuntu Server*, as well as specify the Ubuntu version to be installed (e.g. 20.04)
```

Comment:
```suggestion
2. Execute `./setup-crownlabs-vm.sh <vm-name> create [desktop|server] <ubuntu-version> (--install-guest-additions)` to automatically create a new VM, install the Ubuntu OS and (optionally) the Virtualbox Guest Additions. You need to choose between *Ubuntu Desktop* (i.e. `xubuntu`) and *Ubuntu Server*, as well as specify the Ubuntu version to be installed (e.g. 20.04). In case you select the _server_ version, by default no desktop environment will be installed.
```